core/templates/pages/admin-page/services/admin-router.service.spec.ts

```diff
@@ -68,10 +72,21 @@ describe('Admin router service', () => {
     ars.showTab('#/misc');
     expect(ars.isActivitiesTabOpen()).toBe(false);
     expect(ars.isConfigTabOpen()).toBe(false);
+    expect(ars.isPlatformParamsTabOpen()).toBe(false);
     expect(ars.isRolesTabOpen()).toBe(false);
     expect(ars.isMiscTabOpen()).toBe(true);
   });
 
+  it('should be able to navigate to the platform params tab', () => {
+    expect(ars.isPlatformParamsTabOpen()).toBe(false);
+    ars.showTab('#/platform-parameters');
+    expect(ars.isActivitiesTabOpen()).toBe(false);
+    expect(ars.isConfigTabOpen()).toBe(false);
+    expect(ars.isPlatformParamsTabOpen()).toBe(true);
+    expect(ars.isRolesTabOpen()).toBe(false);
+    expect(ars.isMiscTabOpen()).toBe(false);
```

Comment:
```suggestion
    expect(ars.isPlatformParamsTabOpen()).toBe(false);

    ars.showTab('#/platform-parameters');

    expect(ars.isActivitiesTabOpen()).toBe(false);
    expect(ars.isConfigTabOpen()).toBe(false);
    expect(ars.isPlatformParamsTabOpen()).toBe(true);
    expect(ars.isRolesTabOpen()).toBe(false);
    expect(ars.isMiscTabOpen()).toBe(false);
```

Add a pre-check checking which tab is open.

Reply:
Done